appeals/api/src/database/seed/data-samples.js

```diff
@@ -198,6 +201,18 @@ export const lpaQuestionnaireList = {
 		lpaqCreatedDate: new Date(2023, 4, 9),
 		lpaQuestionnaireSubmittedDate: new Date(2023, 4, 9),
 		isGreenBelt: randomBool()
+	},
+	[APPEAL_TYPE_SHORTHAND_FPA]: {
```

Comment:
I would use the case type code here as it will get complicated when we have more case types.

Reply:
We could change these to use the appeal type codes (eg. D, W) as keys, but that would entail significant refactoring of the seed script, as the `appealFactory` in `appeals/api/src/database/seed/data-test.js` already receives the appeal type shorthand as a parameter, and this is also used to index other objects in the seed data which are also keyed with the shorthand. Passing both the shorthand and the type code would be redundant, and I don't want to get into refactoring the other test code that uses the shorthand in this PR.